index.html

```diff
@@ -209,6 +204,45 @@ <h1 class="main-heading">Pricing</h1>
         </div>
       </div>
 
+
+      <!-- AVAILABLE ON ANY DEVICE SECTION -->
+
+      <div class="container any-device-class">
+        <div class="text-part">
+          <h2 class="text-center any-device-head-text">Access WorkEazee from Any Device</h2>
+          <div class="row text-center"> 
+              <div class="col-lg-4 col-md-4 col-xs-4">
+                <div class="">
+                    <img class="any-device-img" src="images/any-device-image/Laptop_Desktop-removebg.png" alt="Desktop or Laptop">
+                    <h4 class="any-device-text">Desktop / Laptop</h4>
+                </div>
+              </div>
+              <div class="col-lg-4 col-md-4 col-xs-4">
+                <div class="">
+                    <img class="any-device-img" src="images/any-device-image/Mobile-removebg.png" alt="Mobile">
+                    <h4 class="any-device-text">Mobile</h4>
+                </div>
+              </div>
+              <div class="col-lg-4 col-md-4 col-xs-4">
+                <div class="">
+                    <img class="any-device-img" src="images/any-device-image/tablet-removebg.png" alt="Tablet">
+                    <h4 class="any-device-text">Tab / iPad</h4>
+                </div>
+              </div>
+          </div>
+          <div class="text-center">
+            <a href="https://play.google.com/store/apps/details?id=com.aummetals.workeazee">
+              <img class="playstore-logo" src="images/any-device-image/play.png" alt="Play Store Logo">
+            </a>
+          </div>
+          <div class="free-demo-class text-center">
```

Comment:
make this as a different container section

Reply:
Done. Please confirm with the new changes.